app/modules/auth/resources.py

```diff
@@ -209,16 +210,42 @@ def get(self):
         return response
 
 
-@api.route('/code/<string:code_string>')
+@api.route('/code/<string:code_string_dot_json>')
```

Comment:
kinda curious why the `.json` is desirable on this -- but willing to ignore it for the sake of the review. :grin: